abci/server/grpc_server.go

```diff
@@ -62,3 +62,88 @@ func (s *GRPCServer) OnStart(ctx context.Context) error {
 
 // OnStop stops the gRPC server.
 func (s *GRPCServer) OnStop() { s.server.Stop() }
+
+//-------------------------------------------------------
+
+// gRPCApplication is a gRPC shim for Application
+type gRPCApplication struct {
+	app types.Application
+}
+
+func (app *gRPCApplication) Echo(ctx context.Context, req *types.RequestEcho) (*types.ResponseEcho, error) {
```

Comment:
Maybe, for documentation 

```suggestion
func (app *gRPCApplication) Echo(_ context.Context, req *types.RequestEcho) (*types.ResponseEcho, error) {
```